### PR TITLE
audits service : use injection instead of mixins

### PR DESCRIPTION
**Pull request by nonomoho**

### CODE REVIEW

File: generators/client/templates/vue/src/main/webapp/app/components/admin/audits/audits.component.ts.ejs [hunk 1]
```diff
@@ -17,6 +15,7 @@ export default class <%=jhiPrefixCapitalized%>Audits extends mixins(AuditsServic
   public reverse: boolean;
   public toDate: any;
   public totalItems: number;
+  @Inject('auditsService') private auditsService: () => AuditsService;
```
Comment (sahbi-ktifa) [inline generators/client/templates/vue/src/main/webapp/app/components/admin/audits/audits.component.ts.ejs]: Could you move injected field to the top of the class in order to be consistent with other injected files in other components ?

Comment (hdurix) [reply generators/client/templates/vue/src/main/webapp/app/components/admin/audits/audits.component.ts.ejs]: Oops that's my fault, I'm used to have `private` fields declared after the `public` ones.

Comment (nonomoho) [reply generators/client/templates/vue/src/main/webapp/app/components/admin/audits/audits.component.ts.ejs]: We discuss about that with @hdurix, we prefer put private attributes after public.

Comment (sahbi-ktifa) [reply generators/client/templates/vue/src/main/webapp/app/components/admin/audits/audits.component.ts.ejs]: Ok we'll clear this later on another PR :)

**Pull request merged**